docs/migration/migrate_8_0.asciidoc

```diff
@@ -65,7 +65,7 @@ If you are currently using one of these settings in your Kibana config, please r
 ==== Default logging timezone is now the system's timezone
 *Details:* In prior releases the timezone used in logs defaulted to UTC. We now use the host machine's timezone by default.
 
-*Impact:* To restore the previous behavior, in kibana.yml use the pattern layout, with a {kibana-ref}/logging-service.html#date-format[date modifier]:
```

Comment:
I'll add these links back once the PR is merged. We can't link externally to a doc that isn't published yet.